consoles/video_stream.pm

```diff
@@ -202,11 +338,19 @@ sub update_framebuffer ($self) {
     }
 
     # no video connected, don't read anything
-    return 0 unless $self->{ffmpeg};
+    return 0 unless $self->{ffmpeg} or $self->{ustreamer};
 
     my $have_recieved_update = 0;
-    while ($self->_receive_frame()) {
-        $have_recieved_update = 1;
+    if ($self->{ffmpeg}) {
+        while ($self->_receive_frame_ffmpeg()) {
+            $have_recieved_update = 1;
+        }
+    } elsif ($self->{ustreamer}) {
+        # shared-memory interface "discards" older frames implicitly,
+        # no need to loop
+        if ($self->_receive_frame_ustreamer()) {
+            $have_recieved_update = 1;
```

Comment:
the typo was already there in before but I would appreciate if you can fix that in a separate commit in your PR:

```suggestion
            $have_received_update = 1;
```